rest/bulk_api.go

```diff
@@ -423,7 +423,11 @@ func (h *handler) handleBulkDocs() error {
 			err = base.HTTPErrorf(http.StatusBadRequest, "Document body must be JSON")
 			return err
 		}
-		docid, _ := doc["_id"].(string)
+		docid, ok := doc["_id"].(string)
+		if !ok {
+			err = base.HTTPErrorf(http.StatusBadRequest, "Document id must be string")
+			return err
+		}
```

Comment:
I think this is already on master, isn't it? Should this PR just be rebased on master?